### PR TITLE
Update license and attribution

### PR DESCRIPTION
**Pull request by maya**
This updates our license content in the README.md, LICENSE.md, and CONTRIBUTING.md to add info for Bourbon, Neat and Font Awesome icons.

BIG thanks to @brittag for helping out on this one! :star: :fireworks:

### CODE REVIEW

**Comment by maya** [comment]
@mollieru can we update the intro paragraph to closer match what we went with on the homepage. There's some discrepancy between the 508 highest standards language that we went with on the site.

**Comment by mollieru** [comment]
@maya fixed the intro paragraph so to reflect language we're now using on the homepage. thx!

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
+We're so glad you're thinking about contributing to an 18F open source project! If you're unsure about anything, just ask — or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We appreciate all friendly contributions.
 
-Before contributing, we encourage you to read our CONTRIBUTING policy (you are here), our LICENSE, and our README, all of which should be in this repository. If you have any questions, or want to read more about our underlying policies, you can consult the 18F Open Source Policy GitHub repository at https://github.com/18f/open-source-policy, or just shoot us an email/official government letterhead note to [18f@gsa.gov](mailto:18f@gsa.gov).
+We want to ensure a welcoming environment for all of our projects. Our staff follow the [18F Code of Conduct](https://github.com/18F/code-of-conduct/blob/master/code-of-conduct.md) and all contributors should do the same.
```
Comment (maya) [inline CONTRIBUTING.md]: Is it follow or follows?

Comment (maya) [inline CONTRIBUTING.md]: Changed to follow in 4762fd1 after chatting with @awfrancisco

File: LICENSE.md [hunk 2]
```diff
@@ -1,31 +1,55 @@
...
 
-The person who associated a work with this deed has dedicated the work to
-the public domain by waiving all of his or her rights to the work worldwide
-under copyright law, including all related and neighboring rights, to the
-extent allowed by law.
+The person who associated a work with this deed has dedicated the work to the public domain by waiving all of his or her rights to the work worldwide under copyright law, including all related and neighboring rights, to the extent allowed by law.
 
-You can copy, modify, distribute and perform the work, even for commercial
-purposes, all without asking permission.
+You can copy, modify, distribute and perform the work, even for commercial purposes, all without asking permission.
```
Comment (awfrancisco) [inline LICENSE.md]: Our style would put a comma after distribute, but if this is official legal language it's fine to leave it.

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
...
+These tools follow industry-standard web accessibility guidelines and reuse best practices of existing style libraries and modern web design. Created and maintained by [U.S. Digital Service](https://www.whitehouse.gov/digital/united-states-digital-service) and [18F](https://18f.gsa.gov) designers and developers, the Web Design Standards are designed to support government product teams in creating beautiful and easy-to-use online experiences for the American people. Learn more about this project in our announcement [blog post](https://18f.gsa.gov/2015/09/28/web-design-standards/).
```
Comment (awfrancisco) [inline README.md]: "...and reuse _the_ best practices..."

File: LICENSE.md (hunk 2 above)
Comment (maya) [inline LICENSE.md]: Yes in it official language. Will leave as is.

File: README.md (hunk 3 above)
Comment (maya) [inline README.md]: Done in c2fe237.

File: README.md [hunk 4]
```diff
@@ -59,22 +59,32 @@ $ ./go init
...
-Please create a [GitHub Issue](https://github.com/18F/web-design-standards/issues)
+Please create a [GitHub Issue](https://github.com/18F/web-design-standards/issues).
 
-### Public domain
+## Licenses and attribution
 
-This project is in the worldwide [public domain](LICENSE.md). As stated in [CONTRIBUTING](CONTRIBUTING.md):
+### A few parts of this project are not in the public domain
+
+The files in `assets/img` are from [Font Awesome](http://fontawesome.io/) by Dave Gandy. They are under the [SIL Open Font License 1.1](http://scripts.sil.org/OFL).
```
Comment (mollieru) [inline README.md]: Source Sans Pro and Merriweather are also released on SIL Open Font License.  @maya @jpyuda do we also need to list those here? (See language here on font licenses: https://github.com/18F/web-design-standards-assets/pull/9/files)

Comment (maya) [inline README.md]: Let's ask @brittag

Comment (mollieru) [inline README.md]: per @jpyuda 's guidance, just added font license language

Comment (maya) [inline README.md]: @mollieru the license needs to go in LICENSE as well, just like the others.

**Comment by mollieru** [comment]
@maya I don't have experience w this -- can you ping someone to help pull the right part of the license text?

**Pull request merged**